src/lib/util/validateAll.js

```diff
@@ -0,0 +1,21 @@
+
+import assertString from './assertString';
+
+export default function validateAll(value, ...validatorFunctions) {
```

Comment:
It is fairly tricky to pass options.

Reply:
maybe we should create a function in order to bind arguments.
like:
```javascript
    validateAll('a-80-2' ,binder(isPort,'hello',{....) ,isAlphanumeric) // false 
```

Reply:
and also we can use the Currying technique

Reply:
I don't think the `binder` function would be very intuitive  to the average developer. There is probably a more simple approach.